src/app/main/main.component.css

```diff
@@ -22,12 +22,22 @@
 }
 
 .drawer {
-    max-width: 40vw;
+    max-width: 30vw;
     max-height: calc(100vh - 64px);
     overflow: auto;
     flex-shrink: 0;
 }
 
+.sidenav-container {
+    position: fixed;
+    left: 0;
+    right: 0;
+}
```

Comment:
I see that it breaks without this, but using `position: fixed` was unintuitive to me. Is there a link or other explanation we could provide in a comment here?

Reply:
I got this from samples of sidenav. I've only seen position: fixed in the samples. Based on this description it does makes sense to me:

"
position: fixed;
An element with position: fixed; is positioned relative to the viewport, which means it always stays in the same place even if the page is scrolled. The top, right, bottom, and left properties are used to position the element.

A fixed element does not leave a gap in the page where it would normally have been located.

"

Reply:
I did some more experiment. If I remove "display: flex" from .view { ... }, then position: fixed won't be necessary.

Reply:
Ok, that sounds like it might have side effects... this looks good to me!